From f66dcb55aefc3e0427088af958407358ff9c15f2 Mon Sep 17 00:00:00 2001
From: Faisal Hassan <faisal.hassan@oss.qualcomm.com>
Date: Thu, 13 Nov 2025 13:56:45 +0530
Subject: [PATCH 13/17] PENDING: phy: snps-eusb2: Set PHY type for Synopsys
 eUSB2 PHY

Assign PHY_TYPE_USB_EUSB2 to the PHY during probe so that consumers can
identify this PHY type.

Change-Id: I4be58492927dd2cc611e4107cdf8fd7202f8c91d
Signed-off-by: Faisal Hassan <faisal.hassan@oss.qualcomm.com>
---
 drivers/phy/phy-snps-eusb2.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/phy/phy-snps-eusb2.c b/drivers/phy/phy-snps-eusb2.c
index 29badb16a2b2..e46984db5752 100644
--- a/drivers/phy/phy-snps-eusb2.c
+++ b/drivers/phy/phy-snps-eusb2.c
@@ -737,6 +737,8 @@ static int snps_eusb2_hsphy_probe(struct platform_device *pdev)
 		return PTR_ERR(generic_phy);
 	}
 
+	generic_phy->attrs.type = PHY_TYPE_USB_EUSB2;
+
 	dev_set_drvdata(dev, phy);
 	phy_set_drvdata(generic_phy, phy);
 
-- 
2.34.1

